src/web-stack/apps/api/src/app/users/entities/user.entity.ts

```diff
@@ -0,0 +1,43 @@
+import { Entity, PrimaryGeneratedColumn, Column, OneToOne, OneToMany } from 'typeorm';
+import { AffirmationEntity } from './affirmation.entity';
+import { LetterEntity } from './letter.entity';
+import { LikeEntity } from './like.entity';
+import { PhotoEntity } from './photo.entity';
+import { ReaffirmationEntity } from './reaffirmation.entity';
+
+/**
+ * TODO
+ * - Constructor: What should go in there?
+ */
+@Entity({ name: 'user' })
```

Comment:
You can go ahead and define setters for all the fields in the constructor. Do it for all our entities.